interchain-exchange/tutorial/00-tutorial.md

```diff
@@ -351,7 +364,9 @@ BuyOrderBook:
 
 ## Performing an Exchange with a Sell Order
 
-We now have two orders open for MCX: a sell order on the source chain (for 10mcx at 15vcx) and a buy order on the target chain (for 5mcx at 5vcx). Let's perform an exchange by sending a sell order to the source chain.
+<!-- This is a bit difficult to follow. Using a simpler example to illustrate exchange with sell order. Simplifying the calculations by using coherent values for a token and integer amount -->
```

Comment:
before merging, let's delete all comments in the Markdown files because these comments are visible to all since our repo is open source, thanks @alijnmerchant21